docs/installation_guide.md

```diff
@@ -816,160 +877,171 @@ DEFAULT_SILBOPS_BROKER = 'http://pubsub.server.com:8080/silbops/CometAPI'
+from mozilla for more information about how to configure efficiently your https security parameters.
 
 ### Running WireCloud using the Django internal web server
 
-> **NOTE**: Be aware that this way of running WireCloud should be used for evaluation/testing purposes. Do not use it in a production environment.
+> **NOTE**: Be aware that this way of running WireCloud should be used for evaluation/testing purposes. Do not use it in
+> a production environment.
 
 To start WireCloud, type the following command:
 
-    $ python manage.py runserver 0.0.0.0:8000 --insecure
+```bash
+python manage.py runserver 0.0.0.0:8000 --insecure
+```
 
-Then, go to `http://computer_name_or_IP_address:8000/` where computer_name_or_IP_address is the name or IP address of the computer on which WireCloud is installed, and use the username and password you provided when populating the database to sign in on the platform.
+Then, go to `http://computer_name_or_IP_address:8000/` where computer_name_or_IP_address is the name or IP address of
+the computer on which WireCloud is installed, and use the username and password you provided when populating the
+database to sign in on the platform.
 
 ### Running WireCloud using Apache 2
 
-If you choose to deploy WireCloud in Apache, the [mod_wsgi](https://github.com/GrahamDumpleton/mod_wsgi) module must be installed (and so does Apache!).
+If you choose to deploy WireCloud in Apache, the [mod_wsgi](https://github.com/GrahamDumpleton/mod_wsgi) module must be
+installed (and so does Apache!).
 
 #### Installing Apache 2 on Debian/Ubuntu
 
 To do so, type the following command on Debian/Ubuntu:
 
-    $ sudo apt-get install apache2 libapache2-mod-wsgi
+```bash
+sudo apt-get install apache2 libapache2-mod-wsgi
+```
 
 Once you have installed Apache and mod_wsgi you have to enable the latest:
 
-    $ a2enmod wsgi
+```bash
+a2enmod wsgi
+```
 
-The next step is the creation of a `VirtualHost` for WireCloud using the
-Apache's configuration files. On Debian and Ubuntu the configuration files for
-the VirtualHosts are stored at `/etc/apache2/sites-available` and enabled and
-disabled using `a2enmod` and `a2dissite`. The syntax of the `VirtualHost`
-definition depends on the version of Apache 2 of your system, so they are going
-to be described in different sections.
+The next step is the creation of a `VirtualHost` for WireCloud using the Apache's configuration files. On Debian and
+Ubuntu the configuration files for the VirtualHosts are stored at `/etc/apache2/sites-available` and enabled and
+disabled using `a2enmod` and `a2dissite`. The syntax of the `VirtualHost` definition depends on the version of Apache 2
+of your system, so they are going to be described in different sections.
 
 #### Installing Apache 2 on CentOS/RedHat
 
-    $ yum install httpd mod_wsgi
+```bash
+yum install httpd mod_wsgi
+```
 
 #### Apache 2.2
 
 You can use this template as starting point:
 
-```ApacheConf
+```xml
```

Comment:
It would be great if the FIWARE style provides some fixes for the horrendous colours used by `hightlight.js` for the `apacheconf` language. It's the only reason to allow you to use `xml` here, hehe.

Reply:
The FIWARE XML colours are quite harmonious, and work well over soft black: 

![Screenshot 2019-03-26 at 14 24 25](https://user-images.githubusercontent.com/3439249/55004871-eed25900-4fd2-11e9-9bcf-2ecd61a81e0a.png)

And perhaps more importantly  they are consistent.

Reply:
Well, I mean the colours applied when using the `apacheconf` language:


If you find it harmonious, then switch them back to use `apacheconf` that is the correct language (apache config files seems xml files, but are not).

Reply:
Well, I mean the colours applied when using the `apacheconf` language:

<img width="1141" alt="Captura de pantalla 2019-03-26 a las 19 59 42" src="https://user-images.githubusercontent.com/841201/55025715-c6ae1e80-5001-11e9-846b-6952815b5983.png">

I mainly complaint about the red colour, but if you find it harmonious, then switch them back to use `apacheconf` 😉. `apacheconf` is the correct language for apache config files (they are not xml files).

Reply:
Ahhhh - got it. But as you can see (ignoring the red for the moment) it is now using  FIWARE CSS Pink, Yellow and Purple. This means improving the brightness of the red would just  be another CSS change on within the CSS files held by the FIWARE Foundation.

I can look into that after this PR has been processed.  Let's leave it XML for now and potentially change it again later.

Reply:
ok